cli/ballerina-packerina/src/main/java/org/ballerinalang/packerina/utils/DebugUtils.java

```diff
@@ -9,7 +9,7 @@
  */
 public class DebugUtils {
 
-    private static final String DEBUG_ARGS_JAVA8 = "-agentlib:jdwp=transport=dt_socket,server=y,suspend=y";
+    private static final String DEBUG_ARGS_JAVA11 = "-agentlib:jdwp=transport=dt_socket,server=y,suspend=y";
     private static final String JAVA_VERSION_PROP = "java.version";
     private static final String COMPATIBLE_JRE_VERSION = "1.8";
```

Comment:
Do we need to change this to java 11?

Reply:
@daneshk not required for the functionality, just changed along with other changes to make it consistent.
I think line number 14 version also need to be changed 11 @NipunaRanasinghe

Reply:
Yeah line 14 should also be changed and these required changes are already there in https://github.com/ballerina-platform/ballerina-lang/pull/26277. So +1 for getting this merged